lib/twilio-ruby/http/http_client.rb

```diff
@@ -33,7 +33,7 @@ def request(host, port, method, url, params={}, data={}, headers={}, auth=nil, t
         response = @connection.send(method.downcase.to_sym, url, method == 'GET' ? params : data)
 
         @last_response = response
-        if response.body and !response.body.empty?
+        if response.body && !response.body.empty?
```

Comment:
+1

Worth reading: http://www.virtuouscode.com/2010/08/02/using-and-and-or-in-ruby/